tests/sles4sap/redirection_tests/hanasr_schedule_tests.pm

```diff
@@ -47,6 +47,7 @@ sub run {
     connect_target_to_serial(
         destination_ip => $target_node_data{ip_address}, ssh_user => $target_node_data{ssh_user}, switch_root => '1');
 
+    # ( After move to 'lib/sles4sap_publiccloud.pm' this need to be changed to use 'get_hana_topology()' )
```

Comment:
Please remove this comment. 
`get_hana_topology` is a public cloud library based module and won't work with on premise type of test.

Reply:
Done